### DISCUSSION TITLE
Mod suggestion

### DISCUSSION THREAD

**Discussion by Depa9** [ddewar/FS22_UniversalAutoloadModhubAddon#39] [Non-Modhub Mod Support]
Hi!
First of all, sorry, I am new to Git Hub and I don't know if this is the right place for requests.
I thank you for your outstanding work.
I would like to suggest a couple of mods that would be nice to add automatic autoload: 

-Ural_Pack
https://www.kingmods.net/it/fs22/mods/ural-pack

-Scania R730
https://fs22planet.com/it/fs22-scania-r730.html

-Wood Trailer
https://fs22planet.com/it/fs22-wood-trailer.html

Thank you!

**Comment by ddewar** [comment]
Hey there,

First, this is a fine spot for requests, and thank you for letting me know about these mods!

As of right now, I have a few ModHub mods I need to get to first, but I will be sure to try to create some loading zones for these mods. The only downside for these mods is because they are not ModHub mods, I will not be able to add these mods to this mod on modhub, but I will add configurations to this discussion that you can manually add to the UniversalAutodrive.xml to add them for your savegames.

Let me know if you have any questions, and I will keep you updated on the progress of this request.

**Reply by Depa9** [reply]
Oh okay I understand!
I tried configuring the mods myself as instructed, they all seem to work except Ural_Pack which I don't understand why it doesn't recognize the mod (Universal Autoload). Anyway I will try other solutions for the moment!

**Comment by AgHauler** [comment]
I have been trying to figure out how to add Autoload to these two Modhub trailers without much luck. I'm trying to get it to load either 2 stacks of 5's or 2 stacks of 6's. Is this even possible with the way these trailers are built? I have noticed that the TLX trailer will load this length but not correctly. Is this due to the required space in the middle?
Plantation Trailer
https://www.farming-simulator.com/mod.php?lang=en&country=us&mod_id=242524&title=fs2022
Lizard Logger
https://www.farming-simulator.com/mod.php?lang=en&country=us&mod_id=230269&title=fs2022

**Reply by ddewar** [reply]
I'll give this a try and let you know.

**Reply by ddewar** [reply]
This what you're aiming for?

![image](https://user-images.githubusercontent.com/15822357/207220723-706362d8-24d2-4ef1-9ebe-5e2a2b6aab26.png)


``` 
<vehicleConfiguration configFileName="FS22_lizardLogger/lizardLogger.xml" modHubId="230269">
    <loadingArea offset="-0.13 1.86 -0.20" width="2.40" height="2.20" length="13"/>
    <options isLogTrailer="true"/>
</vehicleConfiguration>
```

**Reply by ddewar** [reply]
Not as ideal on the plantation, but works.

![image](https://user-images.githubusercontent.com/15822357/207221523-bcd87942-37f4-48db-b945-9f01e052bf97.png)

```
        <vehicleConfiguration configFileName="FS22_PlantationTrailer/plantationtrailer.xml" modHubId="242524">
            <loadingArea offset="0 1.60 -0.15" width="2.35" height="2.75" length="13"/>
			<options isLogTrailer="true"/>
        </vehicleConfiguration>
```

**Reply by ddewar** [reply]
I have gone ahead and added these to the next update. Thanks for the suggestion.

**Comment by AgHauler** [comment]
Thank you very much for this. They are working in game for me now. Looking through how you are doing this has caused me to understand how a couple of other things I was looking at can be done. I do have one more question though as I am trying to figure out how to add it to another trailer. I think I understand everything except how you come up with the ModHub ID number. How is that done?

**Reply by ddewar** [reply]
Great to hear those are working for you.

I am just pulling those numbers from the modhub link. If you are just doing a local configuration, it is optional and shouldn't cause you any issues.

"https://farming-simulator.com/mod.php?lang=en&country=us&mod_id=260858&title=fs2022" Labeled here as "mod_id"

**Reply by AgHauler** [reply]
I managed to add it to the Hallbilt trailer on Modhub and it works although for some reason it loads and unloads backwards. I'm not real sure why it does this but I'll play around with it and see what happens

**Reply by ddewar** [reply]
I've seen this before with mods that have the translation or something like that backwards. Basically, the game things the front is the back. Easy way to work around it is use ```horizontalLoading="true"``` in the options. Then it might load back to front, but only in 1 layer. That was at least it doesn't pull the front tires off the ground.
